packages/matter.js/src/cluster/server/ClusterServer.ts

```diff
@@ -36,7 +36,7 @@ import { Endpoint } from "../../device/Endpoint.js";
 export type AttributeServers<A extends Attributes> = Merge<Omit<{ [P in MandatoryAttributeNames<A>]: AttributeServer<AttributeJsType<A[P]>> }, keyof GlobalAttributes<any>>, { [P in OptionalAttributeNames<A>]?: AttributeServer<AttributeJsType<A[P]>> }>;
 
 /** Initial values for the cluster attribute */
-export type AttributeInitialValues<A extends Attributes> = Merge<Omit<{ [P in MandatoryAttributeNames<A>]: AttributeJsType<A[P]> }, keyof GlobalAttributes<any>>, { [P in OptionalAttributeNames<A>]?: AttributeJsType<A[P]> }>;
+export type AttributeInitialValues<A extends Attributes> = Merge<{ [P in MandatoryLocalAttributeNames<A>]: AttributeJsType<A[P]> }, { [P in OptionalAttributeNames<A>]?: AttributeJsType<A[P]> }>;
```

Comment:
We cannot change this because this  else destroys typing! And why change it at all? The global attributes are populated internally. No user will ever populate them. 

If you need a Type including them we should use a new type.

Reply:
Wait, is it different semantically?  My intention was just to do the omit and merge in two steps, I was having a hard time parsing.

Reply:
Yes but this is also used e.g. as parameters in constructor which would then pot requitre to also set global attributes that should be explicitely ignored here because populated internally

Reply:
If you look at the definition of MandatoryLocalAttributeNames it's (mandatory - global) so AttributeInitialValues are (mandatory - global + optional).  NBD to do the (mandatory - global) in multiple places though if you think it's safer, I'll maybe just leave a comment that they need to match.

If you ever want spread onto multiple lines with comments I'm happy to volunteer next time I'm digging through. :)

Reply:
Alll ok you added a second one ... then also fine - we just need to check the places where the "former only type" is used and adjust to the new named one! 8at least devcie constructir .. if it stays that way) and also in cluster defs and such